candle-examples/examples/llama/main.rs

```diff
@@ -189,8 +210,18 @@ fn main() -> Result<()> {
             (Llama::load(vb, &cache, &config)?, tokenizer_filename)
         }
     };
+
+    let prompt = match &args.prompt {
```

Comment:
I don't think this `let prompt` bit is actually useful, could you remove it? (in your version, `args.prompt` is checked twice rather than once).